app/components/markdown/markdown.js

```diff
@@ -330,10 +305,14 @@ export default class Markdown extends PureComponent {
             spacer = ' ';
         }
 
+        const parentStyles = this.computeTextStyle(this.props.baseTextStyle, context);
         const style = getStyleSheet(this.props.theme);
         const styles = [
             this.props.baseTextStyle,
-            style.editedIndicatorText
+            style.editedIndicatorText,
+            {
+                fontSize: StyleSheet.flatten(parentStyles).fontSize * 0.6
```

Comment:
Why would the size of this be changing? I'd expect it to be constant even if it's inside a header

Reply:
I actually had changed that when I was trying to figure out the sizing issue.

Reply:
I actually liked it smaller though, now that I switched back. Less obtrusive. ;)